config/200-clusterrole.yaml

```diff
@@ -17,10 +17,22 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: tekton-triggers-admin
 rules:
-  - apiGroups: ["tekton.dev"]
-    resources: ["eventlisteners", "triggerbindings", "triggertemplates"]
```

Comment:
We need access to manipulate the triggers resources `["eventlisteners", "triggerbindings", "triggertemplates"]`.

Also, I think we will be alright using this [`200-clusterrole.yaml`](https://github.com/tektoncd/triggers/blob/master/config/200-clusterrole.yaml) file at first, then adding more resources when it becomes necessary. So, unless the webhook-controller needs access to more resources in order to function, I think we should stick with the current [`200-clusterrole.yaml`](https://github.com/tektoncd/triggers/blob/master/config/200-clusterrole.yaml) file for now.

Reply:
Sorry, my bad for deleting these stuff.
I have check all the configuration, and only remain the necessary ones. thanks.